bin/p2-ds-farm/main.go

```diff
@@ -40,10 +40,10 @@ func main() {
 	quitCh := make(chan struct{})
 
 	_, consulOpts, labeler := flags.ParseWithConsulOptions()
-	client := kp.NewConsulClient(consulOpts)
+	client := consul.NewConsulClient(consulOpts)
 	logger := logging.NewLogger(logrus.Fields{})
 	dsStore := dsstore.NewConsul(client, 3, &logger)
-	kpStore := kp.NewConsulStore(client)
+	consulStore := consul.NewConsulStore(client)
```

Comment:
I want to rename this to `NewStore` since `Consul` is redundant now but i'll save that for later because `gorename` is fighting me